common/common-api/src/main/java/org/ow2/proactive/utils/FileUtils.java

```diff
@@ -169,4 +176,17 @@ public static String getFileNameWithExtension(URL url) {
         return filename;
     }
 
+    public static Properties resolvePropertiesFile(String fileNamePath) {
+        Properties properties = new Properties();
+        try {
+            logger.debug("Loading properties from file " + fileNamePath);
+            FileInputStream stream = new FileInputStream(fileNamePath);
```

Comment:
you can put FileInputStream stream = new FileInputStream(fileNamePath); to the try as it is AutoClosable, I guess..